### PR TITLE
Suggest a <script> block that works for XHTML5, too

### PR DESCRIPTION
**Pull request by tripu**
[Rule #fixupJs](https://www.w3.org/pubrules/doc/rules/?profile=REC#fixupJs)

cf https://github.com/w3c/specberus/pull/297#issuecomment-427820666

/cc @Mouvedia

### CODE REVIEW

**Comment by Mouvedia** [comment]
The most important is to go and fix the specifications that are served as XML.

File: lib/rules.json [hunk 1]
```diff
@@ -1934,7 +1934,7 @@
...
                     "compoundOverview": "The main page <span class=\"rfc2119\">should</span> be called Overview.html.",
                     "compound": "All other files <span class=\"rfc2119\">must</span> be reachable by links from the document.",
-                    "fixupJs": "The document <span class=\"rfc2119\">must</span> include the script <code> <a href=\"https://www.w3.org/scripts/TR/2016/fixup.js\">fixup.js</a></code>.<p><span style=\"font-style: italic\">Include this source code:</span><br/><code>&lt;script src=\"//www.w3.org/scripts/TR/2016/fixup.js\"&gt;&lt;/script&gt;</code></p>"
+                    "fixupJs": "The document <span class=\"rfc2119\">must</span> include the script <code><a href=\"https://www.w3.org/scripts/TR/2016/fixup.js\">fixup.js</a></code>.<p><span style=\"font-style: italic\">Include this source code:</span><br/><code>&lt;script src=\"//www.w3.org/scripts/TR/2016/fixup.js\" type=\"text/javascript\"&gt;&lt;/script&gt;</code></p>"
```
Comment (deniak) [inline lib/rules.json]: According to the [rfc4329](https://tools.ietf.org/html/rfc4329)`text/javascript` is obsolete now. It should be `application/javascript`

Comment (Mouvedia) [reply lib/rules.json]: …which is incompatible with Internet Explorer ≤8.

Comment (tripu) [reply lib/rules.json]: Thanks, @deniak &amp; @Mouvedia; you both taught me something I didn't know.

Anyway, I was already &ldquo;fixing&rdquo; this (see last commit).

[According to caniuse.com](https://caniuse.com/usage-table), all versions of IE &le;8 account for &lt;0.2% of market share. I think IE 8 is now past the point where we should accommodate it or sacrifice features for it&hellip;

That being said, I leave it in the hands of the reviewer to decide whether to take this fix, or revert my last commit :innocent:

**Pull request merged**